cmd/demo/teardown.go

```diff
@@ -24,9 +25,10 @@ Usage
 )
 
 func teardownDemoCluster(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
+	sandboxDefaultConfig := sandboxCmdConfig.DefaultConfig
 	cli, err := docker.GetDockerClient()
 	if err != nil {
 		return err
 	}
-	return sandbox.Teardown(ctx, cli)
+	return sandbox.Teardown(ctx, cli, sandboxDefaultConfig.Verbose)
```

Comment:
I wasn't able to use this config for teardown. I think you are missing the config flag that ties up the command and its default config

```
| Normal | Created           | Created container kubernetes-dashboard                                                           |
+--------+-------------------+--------------------------------------------------------------------------------------------------+
| Normal | Started           | Started container kubernetes-dashboard                                                           |
+--------+-------------------+--------------------------------------------------------------------------------------------------+
| Normal | Pulled            | Successfully pulled image "ecr.flyte.org/bitnami/minio:2021.10.13-debian-10-r0" in 38.115757152s |
+--------+-------------------+--------------------------------------------------------------------------------------------------+
| Normal | Created           | Created container minio                                                                          |
+--------+-------------------+--------------------------------------------------------------------------------------------------+
| Normal | Started           | Started container minio                                                                          |
+--------+-------------------+--------------------------------------------------------------------------------------------------+
| Normal | Pulled            | Successfully pulled image "ecr.flyte.org/ubuntu/postgres:13-21.04_beta" in 1m1.123476844s        |
+--------+-------------------+--------------------------------------------------------------------------------------------------+
| Normal | Created           | Created container postgres                                                                       |
+--------+-------------------+--------------------------------------------------------------------------------------------------+
| Normal | Started           | Started container postgres                                                                       |
+--------+-------------------+--------------------------------------------------------------------------------------------------+
👨‍💻 Flyte is ready! Flyte UI is available at http://localhost:30080/console 🚀 🚀 🎉 
Add KUBECONFIG and FLYTECTL_CONFIG to your environment variable 
export KUBECONFIG=$KUBECONFIG:/Users/praful/.kube/config:/Users/praful/.flyte/k3s/k3s.yaml 
export FLYTECTL_CONFIG=/Users/praful/.flyte/config-sandbox.yaml 
➜  flytectl git:(flyte/#2491) ✗ flytectl demo teardown --verbose
Error: unknown flag: --verbose
ERRO[0000] unknown flag: --verbose  
```